src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -101,14 +102,14 @@ export class ApiKeyAuthenticator {
       let publicKey: Buffer, body: IResponseBody;
 
       if (!StatusCodes.isOk(response.statusCode)
-        && !TypeUtil.isObject(body = response.body) && !body.PublicKey
+        && !TypeUtil.isObject(body = response.body as IResponseBody) && !body.PublicKey
```

Comment:
no, in some cases response is array of objects (for ex DeleteIndex/DatabaseCommand)
- i saw it in python client code..